### PR TITLE
documentation - component lib

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
documentation - component lib - path changed.

### CODE REVIEW

**Review by way2nizam — COMMENTED** [review]
looks good

File: utility/utility.css [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+      }
+
+      @media (max-width: 700px) {
+        .code-snippet-iframe{
+            position: absolute;
+            top: 0;
+            left: 0;
+            height: 100%;
+            /* width: 61.875rem; */
```
Comment (Mohasin-Haque) [inline utility/utility.css:245]: remove commented part

File: utility/utility.css [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 31.25rem;
+  }
+
+
+
+
```
Comment (Mohasin-Haque) [inline utility/utility.css:92]: remove spacing

**Review by Mohasin-Haque — COMMENTED** [review]
Your code is good but try to break PR into more parts, Also remove commented part and whitespacing and format it.

**Pull request merged**